app/templates/home.html

```diff
@@ -3,7 +3,39 @@
 {% set page_title = "Home" %}
 
 {% block main_column %}
+<a href="{{ url_for('apps.list') }}">Apps</a>
 <a href="{{ url_for('users.list') }}">Users</a>
 {% endblock %}
 
-{% block right_column %}&nbsp;{% endblock %}
+{% block right_column %}
+{% if session.passport.user %}
+Hello, {{ session.passport.user.name }}
+<a href="{{ url_for('base.logout') }}">Logout</a>
+{% else %}
+<button id="login">Login</button>
```

Comment:
Couldn't we potentially make this `login` link a link to `/login` and then drop the `login()` function here? Maybe I'm missing something but it seems like it's a duplication of what's happening in that view.

Reply:
(Also if for whatever reason we need to change something login-related we'd then do only in that view)

Reply:
This is meant as an example of different ways of invoking Auth0

Reply:
Yep.